src/main/java/edu/greenblitz/tobyDetermined/commands/LED/ObjectInIntakeLED.java

```diff
@@ -0,0 +1,28 @@
+package edu.greenblitz.tobyDetermined.commands.LED;
+
+import edu.greenblitz.tobyDetermined.OI;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.util.Color;
+
+public class ObjectInIntakeLED extends LEDCommand{
+
+    public ObjectInIntakeLED(){
```

Comment:
IIRC Java creates an empty constructor automatically and you don't need to declare it yourself.